--- rubberband-1.5.0/rubberband.pc.in.fix_ver	2010-05-02 21:06:05.000000000 +0200
+++ rubberband-1.5.0/rubberband.pc.in	2010-07-03 20:01:01.396222596 +0200
@@ -4,7 +4,7 @@
 includedir=${prefix}/include
 
 Name: rubberband
-Version: 1.4.0
+Version: 1.5.0
 Description: 
 Libs: -L${libdir} -lrubberband
 Cflags: -I${includedir} 
--- rubberband-1.5.0/configure.ac.fix_ver	2010-05-02 21:06:05.000000000 +0200
+++ rubberband-1.5.0/configure.ac	2010-07-03 20:02:24.968410068 +0200
@@ -1,5 +1,5 @@
 
-AC_INIT(RubberBand, 1.2, cannam@all-day-breakfast.com)
+AC_INIT(RubberBand, 1.5, cannam@all-day-breakfast.com)
 
 AC_CONFIG_SRCDIR(src/StretcherImpl.h)
 AC_PROG_CXX
