--- rubberband-1.0.1/Makefile.in.destdir	2007-12-13 17:15:03.000000000 -0500
+++ rubberband-1.0.1/Makefile.in	2008-03-30 23:31:44.000000000 -0400
@@ -16,16 +16,16 @@
 
 PROGRAM_TARGET 		:= bin/rubberband
 STATIC_TARGET  		:= lib/librubberband.a
-DYNAMIC_TARGET 		:= lib/librubberband$(DYNAMIC_EXTENSION)
+DYNAMIC_TARGET 		:= lib/librubberband$(DYNAMIC_EXTENSION).1
 VAMP_TARGET    		:= lib/vamp-rubberband$(DYNAMIC_EXTENSION)
 LADSPA_TARGET  		:= lib/ladspa-rubberband$(DYNAMIC_EXTENSION)
 
 INSTALL_BINDIR		:= @prefix@/bin
 INSTALL_INCDIR		:= @prefix@/include/rubberband
-INSTALL_LIBDIR		:= @prefix@/lib
-INSTALL_VAMPDIR		:= @prefix@/lib/vamp
-INSTALL_LADSPADIR	:= @prefix@/lib/ladspa
-INSTALL_PKGDIR		:= @prefix@/lib/pkgconfig
+INSTALL_LIBDIR		:= @libdir@
+INSTALL_VAMPDIR		:= @libdir@/vamp
+INSTALL_LADSPADIR	:= @libdir@/ladspa
+INSTALL_PKGDIR		:= @libdir@/pkgconfig
 
 all:	bin lib $(PROGRAM_TARGET) $(STATIC_TARGET) $(DYNAMIC_TARGET) $(VAMP_TARGET) $(LADSPA_TARGET)
 
@@ -95,7 +95,8 @@
 	$(AR) rsc $@ $^
 
 $(DYNAMIC_TARGET):	$(LIBRARY_OBJECTS)
-	$(CXX) $(DYNAMIC_LDFLAGS) $^ -o $@ $(LIBRARY_LIBS) $(LDFLAGS)
+	$(CXX) -shared -Wl,-soname,librubberband.so.1 $^ \
+		-o $@ $(LIBRARY_LIBS) $(LDFLAGS)
 
 $(VAMP_TARGET):		$(LIBRARY_OBJECTS) $(VAMP_OBJECTS)
 	$(CXX) $(DYNAMIC_LDFLAGS) -o $@ $^ $(VAMP_PLUGIN_LIBS) $(LDFLAGS)
@@ -109,22 +110,24 @@
 	$(MKDIR) $@
 
 install:	all
-	$(MKDIR) -p $(INSTALL_BINDIR)
-	$(MKDIR) -p $(INSTALL_INCDIR)
-	$(MKDIR) -p $(INSTALL_LIBDIR)
-	$(MKDIR) -p $(INSTALL_VAMPDIR)
-	$(MKDIR) -p $(INSTALL_LADSPADIR)
-	$(MKDIR) -p $(INSTALL_PKGDIR)
-	cp $(PROGRAM_TARGET) $(INSTALL_BINDIR)
-	cp $(PUBLIC_INCLUDES) $(INSTALL_INCDIR)
-	cp $(STATIC_TARGET) $(INSTALL_LIBDIR)
-	cp $(DYNAMIC_TARGET) $(INSTALL_LIBDIR)
-	cp $(VAMP_TARGET) $(INSTALL_VAMPDIR)
-	cp src/vamp/vamp-rubberband.cat $(INSTALL_VAMPDIR)
-	cp $(LADSPA_TARGET) $(INSTALL_LADSPADIR)
-	cp src/ladspa/ladspa-rubberband.cat $(INSTALL_LADSPADIR)
+	$(MKDIR) -p $(DESTDIR)$(INSTALL_BINDIR)
+	$(MKDIR) -p $(DESTDIR)$(INSTALL_INCDIR)
+	$(MKDIR) -p $(DESTDIR)$(INSTALL_LIBDIR)
+	$(MKDIR) -p $(DESTDIR)$(INSTALL_VAMPDIR)
+	$(MKDIR) -p $(DESTDIR)$(INSTALL_LADSPADIR)
+	$(MKDIR) -p $(DESTDIR)$(INSTALL_PKGDIR)
+	cp -p $(PROGRAM_TARGET) $(DESTDIR)$(INSTALL_BINDIR)
+	cp -p $(PUBLIC_INCLUDES) $(DESTDIR)$(INSTALL_INCDIR)
+	cp -p $(STATIC_TARGET) $(DESTDIR)$(INSTALL_LIBDIR)
+	cp -p $(DYNAMIC_TARGET) $(DESTDIR)$(INSTALL_LIBDIR)
+	( cd $(DESTDIR)$(INSTALL_LIBDIR) && \
+          ln -s librubberband.so.1 librubberband.so )
+	cp -p $(VAMP_TARGET) $(DESTDIR)$(INSTALL_VAMPDIR)
+	cp -p src/vamp/vamp-rubberband.cat $(DESTDIR)$(INSTALL_VAMPDIR)
+	cp -p $(LADSPA_TARGET) $(DESTDIR)$(INSTALL_LADSPADIR)
+	cp -p src/ladspa/ladspa-rubberband.cat $(DESTDIR)$(INSTALL_LADSPADIR)
 	sed "s,%PREFIX%,@prefix@," rubberband.pc.in \
-	  > $(INSTALL_PKGDIR)/rubberband.pc
+	  > $(DESTDIR)$(INSTALL_PKGDIR)/rubberband.pc
 
 clean:
 	rm -f $(LIBRARY_OBJECTS) $(PROGRAM_OBJECTS) $(LADSPA_OBJECTS) $(VAMP_OBJECTS)
